src/views/SidebarView.vue

```diff
@@ -13,6 +13,11 @@
               :collapsable="true"
               :collapsed="activeAgeFacetId !== 'age'"
               @facetToggle="handleAgeToggle">
+                <template v-slot:label-slot>
+                  <info-icon id="age-info-icon">
+                    Please make a selection if you want to limit your study population to specific age groups. <b-badge pill>optional</b-badge>
```

Comment:
This popup can stay open after the sidebar (select cohorts) is closed, i don't think this is good behavior.

![image](https://user-images.githubusercontent.com/46395349/70141876-3ea40c80-1698-11ea-881c-e2804d917691.png)

Reply:
I think I sort of fixed it for now but the sidebar has other issues too when it gets hidden.
It is not really gone, looks like, just scrolled out of the window. lf you tab around in it, it scrolls back into view.